app-service-template/README.md

```diff
@@ -0,0 +1,41 @@
+# Application Service Template
+
+The contents of this folder is a buildable/runnable template for a new custom application service based on the Hanoi 1.3.1 release of the App Functions SDK. 
+
+> **Note**: If you only need to use the built-in pipeline functions, then it is advisable that you use `App Service Configurable` rather then create a new custom application service. See [here](https://docs.edgexfoundry.org/1.3/microservices/application/AppServiceConfigurable/) for more details on `App Service Configurable`
+
+Do the following to use this template to create your new custom application service:
```

Comment:
How bout "Follow the instructions below to create your new customer application service:".